### PR TITLE
Updated UI and navigation for account creation

### PR DESCRIPTION
**Pull request by abhaykshar**
just made a view UI updates for create account page and set it to navigate to the verify watcard page

### CODE REVIEW

File: android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt [hunk 1]
```diff
@@ -231,29 +228,41 @@ fun CreateAccountPageView(
...
+//			RoundedDropdown(
+//				modifier = Modifier
+//					.fillMaxWidth(ELEMENT_WIDTH),
+//				dropdownItems = Gender.values(),
+//				selectedDropdownItem = uiState.gender,
+//				dropdownItemToString = { stringResource(id = it.stringId)},
+//				setSelectedDropdownItem = {
+//					viewModel.genderStream.onNext(it)
+//				},
+//			)
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageView.kt]: Can you remove this?

**Review by leeminkyu16 — APPROVED** [review]
One small thing.
Let's Gamble, Try Merging

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

**Review by Jiay0928 — APPROVED** [review]
Nice

**Pull request merged**